code/espurna/garland.cpp

```diff
@@ -37,12 +67,33 @@ const char* NAME_GARLAND_SET_BRIGHTNESS = "garland_set_brightness";
 const char* NAME_GARLAND_SET_SPEED      = "garland_set_speed";
 const char* NAME_GARLAND_SET_DEFAULT    = "garland_set_default";
 
-#define EFFECT_UPDATE_INTERVAL_MIN      5000  // 5 sec
-#define EFFECT_UPDATE_INTERVAL_MAX      10000 // 10 sec
+const char* MQTT_TOPIC_GARLAND          = "garland";
+
+const char* MQTT_PAYLOAD_COMMAND        = "command";
+const char* MQTT_PAYLOAD_ENABLE         = "enable";
+const char* MQTT_PAYLOAD_BRIGHTNESS     = "brightness";
+const char* MQTT_PAYLOAD_ANIM_SPEED     = "speed";
+const char* MQTT_PAYLOAD_ANIMATION      = "animation";
+const char* MQTT_PAYLOAD_PALETTE        = "palette";
+const char* MQTT_PAYLOAD_DURATION       = "duration";
+
+const char* MQTT_COMMAND_IMMEDIATE      = "immediate";
+const char* MQTT_COMMAND_RESET          = "reset"; // reset queue
+const char* MQTT_COMMAND_QUEUE          = "queue"; // enqueue command payload
+const char* MQTT_COMMAND_SEQUENCE       = "sequence"; // place command to sequence
```

Comment:
*pedantic note*

`const char* const` is a more correct type here, since we are still able to `MQTT_TOPIC_GARLAND = "blah";` 
(or `const char var[] = ...;`, to const-qualify both at the same time)